lighthouse-cli/test/fixtures/source-maps/bundle-map-500.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+/* eslint-disable */
+
+const a = 1;
+// Use this obnoxious URL because using a non-existent url from localhost will return a
+// bunch of HTML, which will be parsed, and no fetch error occurs.
+//# sourceMappingURL=http://www.this-will-not-exist-blah-go-rockets.com/some-map.js.map
```

Comment:
i guess i could do `http://localhost:1234` ...

Reply:
> go-rockets

😱 

haha 😉 to your actual point though, `:1234` is the default for `parcel` which I tend to have running so that might cause something unexpected too ;) haha 

how about an entirely invalid URL? or is that then testing something different you're not concerned about?

Reply:
i suppose anything that results in a fetch error will work, including a malformed URL.

Reply:
> go-rockets

Did that just to get a rise out of you ;)